src/azure-cli-core/azure/cli/core/_identity.py

```diff
@@ -64,10 +65,13 @@ class Identity:
 
     CLOUD_SHELL_IDENTITY_UNIQUE_NAME = "unique_name"
 
-    def __init__(self, authority=None, tenant_id=None, client_id=None, **kwargs):
+    def __init__(self, authority=None, tenant_id=None, client_id=None, scopes=None, **kwargs):
         self.authority = authority
         self.tenant_id = tenant_id or "organizations"
         self.client_id = client_id or _CLIENT_ID
+        self.scopes = scopes or _DEFAULT_SCOPES
+        if self.scopes and not isinstance(self.scopes, (list, tuple)):
+            self.scopes = (self.scopes,)
```

Comment:
I will unify `scopes` to a `list` as MSAL's `scope` use `list[str]` as the type:

https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/ece3918c62a403b95a078dbccbaf093bc5c6415e/msal/application.py#L523

```py
        :param list[str] scopes: (Required)
            Scopes requested to access a protected API (a resource).
```